include/modem/lte_lc.h

```diff
@@ -464,6 +464,14 @@ enum lte_lc_modem_evt {
 	LTE_LC_MODEM_EVT_OVERHEATED,
 };
 
+/** @brief type of factory reset to perform */
+enum lte_lc_factory_reset_type {
+	/** Reset all modem data to factory settings. */
+	LTE_LC_FACTORYRESET_ALL = 0,
+	/** Reset user-configurable data to factory settings. */
+	LTE_LC_FACTORYRESET_USER = 1,
```

Comment:
I think it would be nicer to separate the words. It's also like this in the enum name.
```suggestion
	LTE_LC_FACTORY_RESET_ALL = 0,
	/** Reset user-configurable data to factory settings. */
	LTE_LC_FACTORY_RESET_USER = 1,
```